mail_connector_queue/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Mail Connector Queue
+====================
+
+This module uses the Connector Queue to send any emails generated by Odoo,
+by creating a queue job for each one.
+
+Configuration
+=============
+
+The cron task `Email Queue Manager` should be disabled, as it conflicts with
+this module.
+
+Usage
+=====
+
```

Comment:
Here (in readme/USAGE.rst), explain that jobs are created automatically (and when) and also explain what the priority field does